torch/ao/quantization/_pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -0,0 +1,205 @@
+        self.global_config = quantization_config
+        return self
+
+    def set_config_for_operator_type(
+        self, operator_type: str, quantization_config: QuantizationConfig
+    ):
+        self.operator_type_config[operator_type] = quantization_config
+        return self
+
+    def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
+        """ just handling global spec for now
+        """
+        model = self._annotate_for_static_quantization_config(model)
+        return model
+
+    def _annotate_for_static_quantization_config(
+        self, model: torch.fx.GraphModule
+    ) -> torch.fx.GraphModule:
+        # annotate the nodes from last to first since the matching is in the reversed order
+        # and fusion operator patterns (conv - relu) can get matched before single operator pattern (conv)
+        # and we will mark the matched node with "_annoated" so fusion operator pattern
+        # can take precedence over single operator pattern in this way
+        config = self.global_config
+        self._annotate_conv2d(model, config)
+
+        return model
+
+    def _annotate_conv2d(
+        self, gm: torch.fx.GraphModule, quantization_config: QuantizationConfig
+    ) -> None:
+        conv_partitions = get_source_partitions(
+            gm.graph, [torch.nn.Conv2d, torch.nn.functional.conv2d]
+        )
+        conv_partitions = list(itertools.chain(*conv_partitions.values()))
+        for conv_partition in conv_partitions:
+            if len(conv_partition.output_nodes) > 1:
+                raise ValueError("conv partition has more than one output node")
+            conv_node = conv_partition.output_nodes[0]
+            if (
+                conv_node.op != "call_function"
+                or conv_node.target != torch.ops.aten.convolution.default
+            ):
+                raise ValueError(f"{conv_node} is not an aten conv2d operator")
+            # skip annotation if it is already annotated
+            if _is_annotated([conv_node]):
+                continue
+            input_qspec_map = {}
+            input_node = conv_node.args[0]
+            assert isinstance(input_node, Node)
+            input_qspec_map[input_node] = get_input_act_qspec(quantization_config)
+
+            weight_node = conv_node.args[1]
+            assert isinstance(weight_node, Node)
+            input_qspec_map[weight_node] = get_weight_qspec(quantization_config)
+
+            bias_node = conv_node.args[2]
+            if isinstance(bias_node, Node):
+                input_qspec_map[bias_node] = get_bias_qspec(quantization_config)
+
+            conv_node.meta["quantization_annotation"] = QuantizationAnnotation(
+                input_qspec_map=input_qspec_map,
+                # TODO<leslie> Remove the annotate of output when oneDNN qconv support fp32 out.
+                output_qspec=get_output_act_qspec(quantization_config),
+                _annotated=True
+            )
```

Comment:
these seems more or less exact copyies of quantizer we have. we were planning to refactor tis into common utils @jerryzh168

Reply:
yeah, I feel this is somewhat tied to quantization_config, we need to figure out the plan for it

Reply:
I think so. One difference is that, we may remove the annotation of `output_qspec` for conv2d in the near future when the backend support it.